qutebrowser/mainwindow/prompt.py

```diff
@@ -422,6 +422,17 @@ def prompt_item_focus(self, which):
         except UnsupportedOperationError:
             pass
 
+    @cmdutils.register(instance='prompt-container', scope='window',
+                       modes=[usertypes.KeyMode.prompt])
+    def prompt_yank(self):
+        """Yank URLs or other data in prompts."""
+        question = self._prompt.question
+        s = None
+        if question and hasattr(question, 'yank_text'):
+            s = question.yank_text
+        utils.set_clipboard(s)
+        message.info("Yanked download URL to clipboard: {}".format(s))
```

Comment:
That seems too specific - you might want to yank other stuff as well. Maybe just `Yanked to clipboard: {}` or so.